### ISSUE TITLE
Enhancement: support normalization options in FFT

### ISSUE THREAD

**Issue by jakevdp** [google/jax#8100]
numpy 1.20 and later supports [several alternate normalizations](https://numpy.org/doc/stable/reference/routines.fft.html#normalization) for FFT functions, including `"backward"`, `"ortho"`, and `"forward"`. We should support these in `jax.numpy.fft` as well.

**Comment by Sudheeradh** [comment]
Hi, I'm a beginner trying to learn scientific computing and open source contribution. I would like to contribute to this issue. 

From what I've seen in the numpy code, the norm is implemented as follows:

<img width="611" alt="Screenshot 2021-10-06 at 1 25 13 AM" src="https://user-images.githubusercontent.com/57762850/136093563-8c78dec9-a19e-4ea6-a565-47065f3454b1.png">

Whereas in JAX code, It is handled as follows in line 34:

<img width="742" alt="Screenshot 2021-10-06 at 1 26 11 AM" src="https://user-images.githubusercontent.com/57762850/136093699-f2551019-6f01-4653-b69a-073b5f6a3877.png">

To solve this is issue, is it sufficient to implement by applying sqrt(n) or n accordingly as in the numpy code or do we need to implement something else? 
I would love to contribute to this issue with help of some guidance. Thanks in advance.

**Comment by Sudheeradh** [comment]
Thanks for the prompt reply. I will try to implement it similar to the numpy code. But, as a beginner I find it quite overwhelming and may take some to do it. I don't want to block anyone else from contributing to this issue, hence, anyone who wants to try this can do so.

**Comment by jakevdp** [comment]
Sure, feel free to open a draft PR and tag me if you have any questions!

**Comment by AniketARS** [comment]
Hi, I've seen this marked as a "good first issue" so this can be a great point to start contributing to `jax`. If anyone working on this already, please tell!
Any help would be great as I have not much experience in open source contribution.

**Comment by jakevdp** [comment]
I haven't seen any work on this functionality yet, so feel free to take a look! Let me know if you have any specific questions.

**Comment by iolloj** [comment]
I've given it a go in #8308
